### PR TITLE
[AN-5123] Show loading dots when downloading images

### PR DESCRIPTION
**Pull request by JPonte**
Ticket: https://wearezeta.atlassian.net/browse/AN-5123
#### APK
[Download build #8678](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8678/artifact/build/artifact/wire-dev-PR728-8678.apk)
[Download build #8687](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8687/artifact/build/artifact/wire-dev-PR728-8687.apk)
[Download build #8692](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8692/artifact/build/artifact/wire-dev-PR728-8692.apk)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -55,7 +55,7 @@ task licenseFormatAndroid(type: nl.javadude.gradle.plugins.license.License) {
 licenseFormat.dependsOn licenseFormatAndroid
 
 ext {
-    majorVersion = "2.30."
```
Comment (hakonbo) [inline build.gradle]: Please revert this change, should still be 2.30

**Comment by JPonte** [comment]
build again
